packages/html/types/index.d.ts

```diff
@@ -0,0 +1,32 @@
+import { Plugin, OutputChunk, OutputAsset, OutputBundle } from 'rollup';
+
+export interface RollupPluginHtmlOptions {
```

Comment:
```suggestion
export interface RollupHtmlOptions {
```

Following existing convention, we don't need `Plugin` in the type names.